File: 9. Planifier.go
package main

import (
	"fmt"
	"os/exec"
)

func main() {
	cmd := `schtasks /Create /SC DAILY /TN TestScript /TR "python C:\Users\Public\TP_Automatisation\1.supprime.py" /ST 02:00 /F`
	err := exec.Command("cmd", "/C", cmd).Run()
	if err != nil {
		fmt.Println("Erreur :", err)
	} else {
		fmt.Println("Tâche planifiée à 2h du matin")
	}
}

File: 9. Planifier.py
import os
task_name = "TestScript"
script_path = r"C:\Users\Public\TP_Automatisation\1.supprime.py"
os.system(f'schtasks /Create /SC DAILY /TN {task_name} /TR "python {script_path}" /ST 02:00 /F')
print("Tâche planifiée à 2h du matin")
